### DISCUSSION TITLE
Known issues with Zim under MacOS 15 ?

### DISCUSSION THREAD

**Discussion by ZimTom** [zim-desktop-wiki/zim-desktop-wiki#2704] [Q&A] [answered]
Dear all,
after the update from MacOS 13 to 14 ended in a midsize desaster I am a bit hesitant this time and use the time until 15.2 is released for checking all used apps for compatibility issues.
Thus I am interested in your opinion / experience with running Zim under MacOS 15.x so far and whether there are known issues.

Thanks in advance for your help.

**Comment by ZimTom** [comment]
Hello René @dehesselle ,
I had hoped to get some reaction from other mac users in the forum. After it remained quiet, I just took the risk and updated to MacOS 15.2 today, because I thought to wait for the fourth update after 15.0 would be safe enough.
The good news is that everything - including heardware-specific stuff - seems to run flawlessly.
But unfortunately it is ZIM that is waving the white flag now.
All I get when I trying to start ZIM is the windows shown below.
Reinstalling the app didn't solve the problem either.

Any help desperately welcome.

Best regards
Thomas

<img width="175" alt="Bildschirmfoto 2024-12-12 um 21 42 44" src="https://github.com/user-attachments/assets/66b5e1fa-3860-470e-844b-0418ce376fc4" />

**Reply by dehesselle** [reply, marked as answer]
This is a known issue and a workaround is available until it's fixed: https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/2665

**Reply by ZimTom** [reply]
René, thanks again for this hint ! Don't know why I didn't find it. Although at hte first glance some of it looks a bit cryptic to me, I'll read through the whole thing to get an understanding of the background and how to apply the workaround.

Looking forward to the result unter-the-hood work and than you for all your effort going in this program.

**Reply by ZimTom** [reply]
Heureka, thanks so much !
It works !
Not that this is a surprise, because if one of the main persons behind the program comes up with something like that, it's the ultimate source. But it is a rrrrreeeeaaaaalllyy nice XMAS present - may be even the best, we'll see :-)

**Comment by dehesselle** [comment]
Asked and answered, closing.
